frontend/src/modules/app/UserActivity.tsx

```diff
@@ -46,13 +71,23 @@ const IdleTimer = ({ timeout, onTimeOut }: { timeout: number; onTimeOut(): void
   return <React.Fragment />;
 };
 
+/**
+ * UserActivity
+ *
+ * @component For tracking the user activity.
+ *
+ * @returns {JSX.Element}
+ */
 export const UserActivity = (): JSX.Element => {
   // 10 year timeout, so that we can temporarily disable the screensaver for v0.7
   const [idleTimeout] = useState(10 * 52 * 7 * 24 * 60 * 60 * 1000);
   // const [idleTimeout] = useState(10 * 60 * 1000);
   const history = useHistory();
   const ventilating = useSelector(getParametersIsVentilating);
 
+  /**
+   * some description
```

Comment:
This is a placeholder which needs to be filled in. i.e. from what you can tell by looking at the code, how is the frontend supposed to behave when the `IdleTimer` times out?

Reply:
On `IdleTimer` timeout, component triggers a callback `onTimeOut` where we can handle what supposed to be done once timed out